mmv1/third_party/terraform/services/networkconnectivity/resource_network_connectivity_group_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package networkconnectivity_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccNetworkConnectivityGroup_BasicGroup(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project_name":            envvar.GetTestProjectFromEnv(),
+		"hub_random_suffix":       acctest.RandString(t, 10),
+		"project_random_suffix_1": acctest.RandString(t, 10),
+		"project_random_suffix_2": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckNetworkConnectivityGroupDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetworkConnectivityGroup_BasicGroup(context),
+			},
+			{
+				ResourceName:            "google_network_connectivity_group.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"hub", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccNetworkConnectivityGroup_BasicGroupUpdate0(context),
+			},
+			{
+				ResourceName:            "google_network_connectivity_group.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"hub", "labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccNetworkConnectivityGroup_BasicGroup(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_network_connectivity_hub" "basic_hub" {
+  name        = "tf-test-hub%{hub_random_suffix}"
+  description = "A sample hub"
+  project     = "%{project_name}"
+  labels = {
+    label-one = "value-one"
+  }
+}
+resource "google_network_connectivity_group" "primary" {
+  hub = google_network_connectivity_hub.basic_hub.id
+  name = "default"
+  auto_accept {
+    auto_accept_projects = ["tf-test-name%{project_random_suffix_1}"]
+  }
```

Comment:
As `auto_accept` is immutable, its change will cause the resource recreation. We don't need the update test for it.

`description` and `labels` fields are mutable. Can you please add the update test for these two fields?

Reply:
ok, done - I guess?